src/osgOcean/OceanScene.cpp

```diff
@@ -21,6 +21,10 @@
 
 #include <osg/Depth>
 
+
+#define SCREEN_WIDTH 1920

```

Comment:
> In this code there are a lot of hard coded parameters, included these image render size. 

There is indeed. But I don't see how that's relevant my specific comment. I did not ask you to move all the hardcoded parameters as arguments.

> To make it a parameter i will need to study Osg Ocean, and now I do not have much time do it.

There's a lot of people around you, some of them having quite a bit of knowledge about osgocean. You could have tried to do the right thing, and asked !